eq-author/src/components/EditorLayout/Tabs/index.js

```diff
@@ -73,9 +90,14 @@ export const UnwrappedTabs = props => {
                 otherProps: { to: url(match), activeClassName },
               }
             : { Component: DisabledTab };
-
           return (
             <Component data-test={key} key={key} {...otherProps}>
+              {key === "design" &&
```

Comment:
Going to have an issue once we have logic page errors are included in with the page errors. 
Will have to filter the page errors in some fashion,